bower.json

```diff
@@ -21,7 +21,8 @@
   "ignore": [],
   "dependencies": {
     "polymer": "Polymer/polymer#^1.0.0",
-    "iron-ajax": "PolymerElements/iron-ajax#^1.0.0"
+    "iron-ajax": "PolymerElements/iron-ajax#^1.0.0",
+    "paper-spinner": "PolymerElements/paper-spinner#~1.0.3"
```

Comment:
This is a `dev` dependency, since it's used in the demo and not the element itself :)